apps/api/src/server/utils/mapping/map-application-with-sector-and-subsector.js

```diff
@@ -19,8 +19,8 @@ export const mapApplicationWithSectorAndSubSector = (application) => {
 
 	return {
 		...applicationData,
-		subSector: mapSector(application.ApplicationDetails.subSector),
-		sector: mapSector(application.ApplicationDetails.subSector.sector),
```

Comment:
to the backendTeam:  this was necessary to prevent draft applications with no sectors to destroy everything.
this way the api returns sector and subsectors null and the FE filters them by status